### ISSUE TITLE
Keyboard shortcut of "New job"

### ISSUE THREAD

**Issue by bertfrees** [daisy/pipeline-ui#35]
There are two issues with the keyboard shortcut for "New job":

- The key combination is not indicated in File menu or tray menu.
- The current key combination is Control + Opt + N (according to the README). Perhaps we can change it to something more standard like ⌘N or ⌘T (on macOS)?

**Comment by marisademeglio** [comment]
Yes we need to better integrate the keyboard shortcuts, this was a fast way to add them and hopefully get some initial feedback. 

Both this and #7 would be best addressed via #32 (would introduce commands that can be triggered from anywhere).

**Comment by NPavie** [comment]
will be fixed by 05d4a3ca
